subprojects/docs/src/samples/userguide/dependencyManagement/definingUsingConfigurations/inheritance/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    `java-library`
+}
+
+repositories {
+    mavenCentral()
+}
+
+// tag::configuration-definition[]
+val smokeTest by configurations.creating {
+    extendsFrom(configurations["testImplementation"])
+}
+
+dependencies {
+    testImplementation("junit:junit:4.12")
+    smokeTest("org.apache.httpcomponents:httpclient:4.5.5")
+}
+// end::configuration-definition[]
+
+task("copyLibs", Copy::class) {
```

Comment:
:x: `task<Copy>("copyLibs") {`

Reply:
Hmm.. I've been telling folks to use `task("copyLibs", Copy::class)`, but your suggestion is cleaner. Guess we have some other samples to go fix.